app/src/main/java/com/pajato/android/gamechat/fragment/GameFragment.java

```diff
@@ -1,30 +1,146 @@
+/*
+ * Copyright (C) 2016 Pajato Technologies LLC.
+ *
+ * This file is part of Pajato GameChat.
+
+ * GameChat is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+
+ * GameChat is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * General Public License for more details.
+
+ * You should have received a copy of the GNU General Public License along with GameChat.  If not,
+ * see http://www.gnu.org/licenses
+ */
 package com.pajato.android.gamechat.fragment;
 
 import android.os.Bundle;
+import android.support.design.widget.Snackbar;
 import android.support.v4.app.Fragment;
 import android.view.LayoutInflater;
 import android.view.View;
 import android.view.ViewGroup;
 
 import com.pajato.android.gamechat.R;
 
+import java.util.ArrayList;
 
+/**
+ *
+ *
+ * @author Bryan Scott -- bryan@pajato.com
+ */
 public class GameFragment extends Fragment{
 
+    /* The Match History */
+    private ArrayList<String> mInstructions;
+    /* Keeps track of the mTurn user. True = Player 1, False = Player 2. */
+    private boolean mTurn;
+    /* Player Turn Strings */
+    private String mPLAYER1;
+    private String mPLAYER2;
+
+    // Game Fragments
+    /* The Tic-Tac-Toe fragment */
```

Comment:
At some point, the GameFragment class does not want to have any awareness of the actual game being played.  We want to be able to add new games without making any changes to GameFragment.